sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ManagedFederatedIdentityCredential.cs

```diff
@@ -44,71 +44,8 @@ internal class ManagedFederatedIdentityCredential : TokenCredential
         ///         </item>
         ///       </list>
         /// </param>
-        /// <param name="additionallyAllowedTenants">The set of </param>
-        public ManagedFederatedIdentityCredential(string tenantId, string clientId, string managedIdentityId, string azureCloud, IEnumerable<string> additionallyAllowedTenants = default)
```

Comment:
I see what you're up to.   Nice refactor, shifting the identity creation up to where we already have to deal with the different combinations!